README.md

```diff
@@ -1,185 +1,164 @@
-
-```
-brew install python3
-brew postinstall python3
-```
+We use strict flake8, pylint and black linters to check the validity of our code.
 
+For the backend, run:
 
-### The virtualenv
+    $ make lint-back
 
-Place yourself in the application directory `app`:
+For the frontend, run:
 
-    cd app
+    $ make lint-front
 
-We choose to run our application in a virtual environment.
 
-For this, we'll install `virtualenvwrapper` and add an environment:
+### Running tests
 
-    pip install virtualenvwrapper
+For the backend, run:
 
-You can open a new shell to activate the virtualenvwrapper commands, or simply do:
+    $ make test-back
 
-    source $(which virtualenvwrapper.sh)
+For the frontend, run:
 
-Then create the virtual environment for `richie`:
+    $ make test-front
 
-    mkvirtualenv richie --no-site-packages --python=python3
+### Running migrations
 
-The virtualenv should now be activated and you can install the Python dependencies for development:
+The first time you start the project with `make bootstrap`, the `db` container automatically
+creates a fresh database named `richie` and performs database migrations. Each time a new
+**database migration** is added to the code, you can synchronize the database schema by running:
 
-    pip install -r requirements/dev.txt
-
-The "dev.txt" requirement file installs packages specific to a dev environment and should not be used in production.
+    $ make migrate
 
 
 ### Settings
 
-Settings are defined in different files for each of the following environments:
-
-- dev: settings for development on developpers' local environment,
-- ci: settings used on the continuous integration platform,
-- staging: settings for deployment to the staging environment,
-- preprod: settings for deployment to the pre-production environment,
-- prod: settings for deployment to the production environment.
-
-The `dev` environment is defined as the default environment.
-
-For development, you can add your own settings file named `local.py` to customize settings to your needs. It will be ignored by git.
-
-### Frontend build
-
-This projet is a hybrid that uses both Django generated pages and frontend JS code. As such, it includes a frontend build process that comes in two parts: JS & CSS.
-
-We need NPM to install the dependencies and run the build, which depends on a version of Nodejs specified in `.nvmrc`. See [the repo](https://github.com/creationix/nvm) for instructions on how to install NVM. To take advantage of `.nvmrc`, run this in the context of the repository:
-
-    nvm install
-    nvm use
-
-As a prerequisite to running the frontend build for either JS or CSS, you'll need to [install yarn](https://yarnpkg.com/lang/en/docs/install/) and download dependencies _via_:
-
-    yarn install
+Settings are defined using Django Configuration for different environments:
```

Comment:
Maybe add a link to Django-Configurations project: https://github.com/jazzband/django-configurations/